app/lib/logic/rolling_updates.rb

```diff
@@ -271,17 +271,7 @@ def missing_config
         end
       end
 
-      class Policies < Base
-        def missing_config
-          false
-        end
-      end
-
       class PolicyRegistry < Base
-        def enabled?
-          super && provider_can_use?(:policies)
```

Comment:
I would be conservative here. We decide to enable it for everybody. Just make it explicit we are enabling it, then in another PR we can just remove it, making sure that `provider_can_use.*(:policies)` and `provider_can_use.*(:policy_registry)` are all removed from the code.

Instead I would suggest:

```ruby
  def enabled?
    true
  end

  def missing_config
     true
  end
```